go.mod

```diff
@@ -2,48 +2,37 @@ module github.com/pingcap/tidb
 
 require (
```

Comment:
why this file changed?

Reply:
i don't know. just changed after `make dev`